packages/common-utils/src/renderChartConfig.ts

```diff
@@ -816,36 +816,80 @@ function translateMetricChartConfig(
       whereLanguage: 'sql',
     };
   } else if (metricType === MetricsDataType.Sum && metricName) {
+    const time_bucket_col = '`__hdx_time_bucket2`';
+    const value_high_col = '`__hdx_value_high`';
+    const value_high_prev_col = '`__hdx_value_high_prev`';
+    const time_expr = timeBucketExpr({
```

Comment:
style: we use camel-case across the ts codebase :)

Reply:
Sorry, my python/rust habits surface.